tests/integration/test_data_row_metadata.py

```diff
@@ -17,7 +18,7 @@
 TEXT_SCHEMA_ID = "cko8s9r5v0001h2dk9elqdidh"
 CAPTURE_DT_SCHEMA_ID = "cko8sdzv70006h2dk8jg64zvb"
 PRE_COMPUTED_EMBEDDINGS_ID = 'ckrzang79000008l6hb5s6za1'
-CUSTOM_TEXT_SCHEMA_NAME = 'custom_text'
+CUSTOM_TEXT_SCHEMA_NAME = 'datarow_metadata_custom_text'
```

Comment:
I wonder if we want to use random schema name here for each test?

Reply:
I think this change is not needed anymore so I'll revert it, before we were creating and deleting this for each test, but that doesn't work with parallel tests.